eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -10,21 +10,27 @@ parameters:
     Linux_Node10:
       OSVmImage: "ubuntu-16.04"
       TestType: "node"
+      RunCodeCoverage: true
     Windows_Node10:
       OSVmImage: "windows-2019"
       TestType: "node"
+      RunCodeCoverage: false
```

Comment:
I would use a global variable or a variable defined in the scope of the job that sets a default value for `RunCodeCoverage` ... Since there are 64 instances of `RunCodeCoverage: false` and 20 instances of `RunCodeCoverage: true` we can save the most lines of code by defaulting to `false`

Reply:
done